pkg/kubelet/dockershim/docker_service.go

```diff
@@ -130,7 +132,7 @@ func NewDockerService(client dockertools.DockerInterface, seccompProfileRoot str
 	cniPlugins = append(cniPlugins, kubenet.NewPlugin(pluginSettings.PluginBinDir))
 	netHost := &dockerNetworkHost{
 		pluginSettings.LegacyRuntimeHost,
-		&namespaceGetter{ds},
```

Comment:
why not just add a hostportGetter? 
The point of composing the network host from a bunch of interfaces is that it makes using a different implementation easier.